department-of-veterans-affairs/caseflow
# Making Reader work with AMA appeals

Connects #5731

### Description
This PR makes three changes:
1) Changes reader to get either a legacy or AMA appeal based on the id
2) Changes document service code to reference veteran_file_number instead of sanitized_vbms_id since that naming is more consistent between legacy and AMA
3) Makes Reader use the new appeal serializers from Queue
4) Changes some parts of the frontend to work with the AMA appeals

### Acceptance Criteria 
- [ ] Can access reader from `queue/beaam` page

### Testing Plan
1. Run `rake db:seed`
1. Go to `queue/beaam` with the `:queue_beaam_appeals` feature toggle.
1. Click on a reader link and ensure you can get to reader.
